includes/embeds/class-amp-twitter-embed.php

```diff
@@ -229,15 +245,27 @@ private function create_amp_twitter_and_replace_node( $dom, $node ) {
 			return;
 		}
 
+		$additional_attributes = [];
+
+		foreach ( $this->supported_data_attributes as $attr ) {
```

Comment:
As opposed to limiting the attributes to just the explicitly supported ones, why not just copy _everything_? Any unrecognized `data-*` attributes will just be ignored by `amp-twitter`, but they may have some other purpose on the original `blockquote` element that had them.

Reply:
Yes I can re-work it on that basis.